Configuration/StandardSequences/python/Eras.py

```diff
@@ -56,7 +58,7 @@ def __init__(self):
         self.Run2_HI = cms.ModifierChain( self.run2_common, self.run2_HI_specific, self.stage1L1Trigger )
         # Future Run 2 scenarios.
         self.Run2_2016 = cms.ModifierChain( self.run2_common, self.run2_25ns_specific, self.stage2L1Trigger, self.run2_HF_2016 )
-        self.Run2_2017 = cms.ModifierChain( self.Run2_2016, self.phase1Pixel, self.trackingPhase1 )
+        self.Run2_2017 = cms.ModifierChain( self.Run2_2016, self.phase1Pixel, self.trackingPhase1, self.muonTrigger2017 )
```

Comment:
@dildick Please add `muonTrigger2017` also to `Run2_2017_trackingPhase1PU70` and `Run2_2017_trackingRun2` below at lines 70-71.

@Dr15Jones I think `ModifierChain.copyAndExclude()` or similar would start to be useful.